Subject: [PATCH] ipr: fix compile failure
From: Christoph Hellwig <hch@lst.de>
Git-commit: d247a70a4d67623eb738176ee703a7bb8b6c325c
Patch-mainline: v3.17-rc5-180-gd247a70

Fix a typo in the IPR_IOASC_HW_CMD_FAILED declaration.  Based on a patch
from Wen Xiong <wenxiong@linux.vnet.ibm.com>.

Signed-off-by: Christoph Hellwig <hch@lst.de>
Acked-by: Dinar Valeev <dvaleev@suse.com>
---
 drivers/scsi/ipr.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: b/drivers/scsi/ipr.h
===================================================================
--- a/drivers/scsi/ipr.h	2014-12-10 02:42:43.061743023 -0600
+++ b/drivers/scsi/ipr.h	2014-12-10 02:43:32.041743002 -0600
@@ -130,7 +130,7 @@
 #define IPR_IOASC_HW_DEV_BUS_STATUS			0x04448500
 #define	IPR_IOASC_IOASC_MASK			0xFFFFFF00
 #define	IPR_IOASC_SCSI_STATUS_MASK		0x000000FF
-#define IPR_IOASC_HW_CMD_FAILEd			0x046E0000
+#define IPR_IOASC_HW_CMD_FAILED			0x046E0000
 #define IPR_IOASC_IR_INVALID_REQ_TYPE_OR_PKT	0x05240000
 #define IPR_IOASC_IR_RESOURCE_HANDLE		0x05250000
 #define IPR_IOASC_IR_NO_CMDS_TO_2ND_IOA		0x05258100
